config/build/dpkg_build/set_robotenv.sh

```diff
@@ -8,4 +8,19 @@ export RobotToolsPath=/opt/rfwaio/tools
 export RobotTestPath=~/RobotTest/testcases
 export RobotLogPath=~/RobotTest/logfiles
 export RobotTutorialPath=~/RobotTest/tutorial
-export GENDOC_PLANTUML_PATH=$RobotVsCode/data/extensions/jebbs.plantuml-2.17.5
\ No newline at end of file
+export GENDOC_PLANTUML_PATH=$RobotVsCode/data/extensions/jebbs.plantuml-2.17.5
+
+
+# Check if JAVA_HOME is already set
+if [ -z "$JAVA_HOME" ]; then
+    # Find the path of OpenJDK using the update-alternatives command
+    JDK_PATH=$(update-alternatives --query javac | grep 'Value: ' | grep -o '/.*/bin/java')
+    
+    if [ -n "$JDK_PATH" ]; then
+        # Export JAVA_HOME if OpenJDK path is found
+        export JAVA_HOME=$(dirname $(dirname $JDK_PATH))
+        echo "JAVA_HOME set to $JAVA_HOME"
+    else
+        echo "No OpenJDK installation found."
```

Comment:
Hi Thong,
Please remove this this message (No OpenJDK installation found). This is only confusing, but OpenJDK is for normal users not required.
JAVA_HOME set to ... is O.K.
Thank you,
 Thomas